.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I agree in the sense that it's not specific to TS. On the other hand, there's some stuff here that makes developing in it way easier (and I know quite a few of us are now using it).
Need more opinions from others, please!